protocol-spec/generic.md

```diff
@@ -134,5 +142,150 @@
 ]
 ```
 
+## VibrateCmd
+
+**Description:** Causes a toy that supports vibration to run specific vibration motors at a certain speeds. In order to abstract the dynamic range of different toys, the values are sent as a dictionary of vibration motor indexes against floats with a range of \[0.0-1.0\]
```

Comment:
Ok, I'm not happy with my own wording on a lot of these message descriptions. The "dynamic range" thing is too complex, and there's no reason to say what's in a message right before we show the layout of it, which also has the definition. I can see you're mainly using what I already had there. I'm gonna file a followup for myself to go back and change the rest of the messages, as well as reword some other stuff, but for now, hear me out on this wording, here and on the other comments below. 

Though, just an update too: Everything you call dictionary, here and below, seems to actually be an array. :)

Rewording:

Causes a toy that supports vibration to run specific vibration motors at a certain speeds. Devices with multiple vibrator features may take multiple values. The FeatureCount attribute for the message in the DeviceList/DeviceAdded message will contain that information.